magento/magento2
# Wrong type hint for the protected member $_urlHelper

The type hint for the protected member `$_urlHelper` was wrong.
Undefined class 'Helper':95

### Description (*)
Fixes the wrong type hint with the correct one, that is also used in constructor

### Related Pull Requests

### Fixed Issues (if relevant)


### Manual testing scenarios (*)
- Open the file in PHPStorm
- See the warning

### Questions or comments
The issue does not have any negative effect to my knowledge, so this is considred a minor issue.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#32503: Wrong type hint for the protected member $_urlHelper
